nix/ouroboros-network.nix

```diff
@@ -36,9 +36,26 @@ let
     modules = [
 
       {
-        # Compile all local packages with -Werror:
+        # Compile all local packages with -Werror and only
```

Comment:
I suspect the rest of the Consensus team would also like to enable `--Wunused-packages` everywhere. Perhaps we can simplify this Nix by just setting the flag for all packages instead of just `-network` packages?

Reply:
I think we should set these flags in the respective `.cabal` files. Only `-Werror` is added via the Nix infra for CI as it is very inconvenient to have it enabled when developing locally.

Reply:
I moved the `-Wunused-packages` to `*.cabal` files and removed this `nix` change.